androidshared/src/main/java/org/odk/collect/androidshared/ui/PrefUtils.kt

```diff
@@ -0,0 +1,83 @@
+package org.odk.collect.androidshared.ui
+
+import android.content.Context
+import androidx.preference.ListPreference
+import org.odk.collect.shared.settings.Settings
+import java.util.Arrays
+
+object PrefUtils {
+
+    @JvmStatic
+    fun createListPref(
+        context: Context,
+        key: String,
+        title: String,
+        labelIds: IntArray,
+        values: Array<String>,
+        settings: Settings,
+    ): ListPreference {
+        val labels = arrayOfNulls<String>(labelIds.size)
```

Comment:
`arrayOfNulls`? why?

Reply:
It's what Kotlin conversion spat out for [this line](https://github.com/getodk/collect/blob/03137bfd820fc4ec88baf1f963c6855a0fad968c/collect_app/src/main/java/org/odk/collect/android/preferences/PrefUtils.java#L22) which does make sense I guess. I'll clean this class up though, so it's less obviously just a straight Java -> Kotlin conversion.